source/classes.tex

```diff
@@ -4548,8 +4542,8 @@
 \end{note}
 The access to a member is affected by the class in which the member is
 named.
-This naming class is the class in which the member name was looked
-up and found.
+This naming class is
+the class in whose scope name lookup performed a search that found the member.
```

Comment:
"whose" -> "which" or "in whose" -> "whose"

Reply:
"in whose" feels right to me

Reply:
exactly, that's the existing wording :)
I proposed "whose" -> "which"
or "in whose" -> "whose",
either works.

BTW, just found another instance of "in whose" in [module.unit]/p7.3 that should be fixed also, hmm.

Reply:
I understand your suggestion, but I'm not convinced it's an improvement. "in whose" seems a good genitive to me (caveat: not a native speaker). "This naming class is the class in the scope of which name lookup performed a search..." is the meaning, but sounds non-sophisticated to me. "of which" -> "whose" seems a good replacement; that leaves the "in".

Reply:
I tend to agree that the current wording is the better of the two. How about shortening the last part, though, perhaps:

"""
The naming class is the class in whose scope name lookup succeeded by finding the member.
"""

(I'm tempted to relegate either, or any, rewrite to a separate PR, though.)

Reply:
Deferred.